manifests/auth_param.pp

```diff
@@ -14,7 +14,7 @@
   concat::fragment{"squid_auth_param_${auth_param_name}":
     target  => $::squid::config,
     content => template('squid/squid.conf.auth_param.erb'),
-    order   => "40-${order}-${scheme}",
+    order   => "05-${order}-${scheme}",
```

Comment:
It does! I arbitrarily picked `05` because it's less than `10` but we could renumber them all if you like.